unaligned_bam_to_bqsr_and_vcf/bwa_mem.cwl

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+label: "run bwa_mem"
+baseCommand: ["/usr/local/bin/bwa", "mem"]
+arguments: 
+    ["-K", "10000000",
+    "-t", "8",
+    "-Y"]
+requirements:
+    - class: DockerRequirement
+      dockerPull: "registry.gsc.wustl.edu/genome/bwa-0.7.15:1"
+stdout: 'refAlign.sam'
+inputs:
+    readgroup:
+        type: string
+        inputBinding:
+            prefix: "-R"
+            position: 1
+    reference:
+        type: File
+        inputBinding:
+            position: 2
+        secondaryFiles: [.fai, .bwt, .sa, .ann, .amb, .pac]
```

Comment:
Same comment as earlier, BWA-MEM will want the .alt secondaryFile to avoid discounting mapping score when reads align to alternate haplotypes.